pyxem/signals/diffraction2d.py

```diff
@@ -614,7 +614,12 @@ def get_direct_beam_position(self, method, lazy_output=None, **kwargs):
         Parameters
         ----------
         method : str,
-            Must be one of "cross_correlate", "blur" or "interpolate"
+            Must be one of "cross_correlate", "blur", "interpolate" or "center_of_mass".
+
+           "cross_correlate": Center finding using cross-correlation of circles of `radius_start` to `radius_finish`.
+           "blur": Center finding by blurring each frame with a Gaussian kernel with standard deviation `sigma` and finding the maximum.
+           "interpolate": Finding the center by summing along X/Y and finding the peak for each axis independently. Data is blurred first using a Gaussian kernel with standard deviation "sigma".
+           "center_of_mass": The center is found using a calculation of the center of mass. Optionally a `mask` can be applied to focus on just the center of some dataset.
```

Comment:
The center of mass function also has a threshold parameter, which can help to suppress the contrast from weaker diffraction features.